BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Internal/Tracer.cs

```diff
@@ -102,7 +113,37 @@ public void OnSpanEnd(Span span)
                 }
                 return;
             }
-            Sample(span);
+            else
+            {
+                RunOnEndCallbacks(span);
+            }
+            if (!span.WasAborted)
+            {
+                Sample(span);
+            }
+        }
+
+        public void RunOnEndCallbacks(Span span)
+        {
+            if (!span.WasAborted && _onSpanEndCallbacks != null)
+            {
+                foreach (var callback in _onSpanEndCallbacks)
+                {
+                    try
+                    {
+                        if (!callback.Invoke(span))
+                        {
+                            span.Abort();
+                            break;
+                        }
+                    }
+                    catch (Exception e)
+                    {
+                        Debug.LogError("Error running OnSpanEndCallback: " + e.Message);
+                    }
+                }
```

Comment:
Maybe I've missed it, but I would expect to see the `bugsnag.span.callbacks_duration` implementation here?